azure-storage-blob/azure/storage/blob/baseblobservice.py

```diff
@@ -1534,6 +1534,11 @@ def set_blob_service_properties(
         :type static_website:
             :class:`~azure.storage.common.models.StaticWebsite`
         '''
+        if all(parameter is None for parameter in [logging, hour_metrics, minute_metrics, cors, target_version, timeout,
+                                                   delete_retention_policy, static_website]):
+
+            raise ValueError("set_blob_service_properties should be called with at least one non-None parameter")
```

Comment:
minor: no need for non-None, "at least one parameter" implies non-None already

Reply:
This is addressed in the second commit